src/styles/images/index.md.njk

```diff
@@ -4,26 +4,60 @@ description: Only use images if there’s a real user need
 section: Styles
 backlog_issue_id: 70
 layout: layout-pane.njk
+show_page_nav: true
 ---
 
 {% from "_example.njk" import example %}
 
 Avoid unnecessary decoration. Only use images if there’s a real user need.
 
-## Aspect ratio
+Make sure any information contained in an image can be understood by someone who cannot see it.
 
```

Comment:
Can expand on the intro lines with this bit lifted from [alt text edits](https://docs.google.com/document/d/1c8axV52ECFzn077hcMwlHfMOobQBRLpTEHy8v0d8-Kw/edit?usp=sharing).

```suggestion
## Use images to give information

Services usually work best without relying on images. Focus on writing clear, simple content for your service first.
Adding images can sometimes help users better understand instructions. However, images should only be visual aids — written content must contain all the information the user needs to complete the service.

```